test/nextjournal/markdown_test.cljc

```diff
@@ -1013,6 +1013,37 @@ back to text") :content second)))
 ```")
                  :content first))))))
 
+(deftest html-test
+  (is (match? [{:type :paragraph}
+               {:type :html-block, :text #"<article>\nfoobar\n</article>\s?"}
+               {:type :paragraph}]
+              (-> (md/parse "Hello
+
+<article>
+foobar
+</article>
+
+Bye") :content)))
+  (is (match? [{:type :paragraph}
+               {:type :html-block, :text #"<img src=\"foo\"/>\s?"}
```

Comment:
@zampino The `\s?` is because of markdown-it adding a newline after the content whereas commonmark does not.